compose-builder/tui-generator.sh

```diff
@@ -0,0 +1,293 @@
+            Press <ENTER> to Skip \n
+            "
+
+    # Generate a Specific form of Array required by whiptail checklist
+    # FORMAT: "<INDEX> <DESCRIPTION> <OFF>"
+    arglist=()
+    for index in "${additionalOpts[@]}";
+    do
+        arglist+=("$index")
+        arglist+=("${additionalOptsDesc[$index]}")
+        arglist+=("OFF") # Nothing is selected by-default
+    done
+    SELECTED_OTHERS+=$(/usr/bin/whiptail --title "App Services" \
+                --notags --separate-output \
+                --ok-button Next \
+                --nocancel \
+                --checklist "$message" 20 75 5 \
+                -- "${arglist[@]})" \
+                3>&1 1>&2 2>&3)
+}
+
+
+####################################################################
+#    App Services Options Display Function                         #
+####################################################################
+function appServiceOption() {
+    message="Available App Services to include in your compose file. \n
+            Press <SPACEBAR> to Select \n
+            Press <ENTER> to Skip \n
+            "
+
+    # Generate a Specific form of Array required by whiptail checklist
+    # FORMAT: "<INDEX> <DESCRIPTION> <OFF>"
+    arglist=()
+    for index in "${appServices[@]}";
+    do
+        arglist+=("$index")
+        arglist+=("${appServiceDesc[$index]}")
+        arglist+=("OFF") # Nothing is selected by-default
+    done
+    SELECTED_APPSERVICES+=$(/usr/bin/whiptail --title "App Services" \
+                --ok-button Next \
+                --nocancel \
+                --notags --separate-output \
+                --checklist "$message" 20 75 7 \
+                -- "${arglist[@]}" \
+                3>&1 1>&2 2>&3)
+}
+
+
+
+####################################################################
+#    ARM64 Question Display Function                               #
+####################################################################
+function displayArm64() {
+    message="Would you like to use ARM64 Images to generate the Compose file?"
+
+    /usr/bin/whiptail --title "Images Architecture" --yesno "$message" 8 78
+}
+
+
+####################################################################
+#    No-Security Question Display Function                         #
+####################################################################
+function displayNoSecty() {
+    message="Would you like to generate a non-secure configuration of the Compose file?"
+    
+    /usr/bin/whiptail --title "Non-Secure Configuration" --yesno "$message" 8 78
+}
+
+
+####################################################################
+#    Device Services Options Display Function                      #
+####################################################################
+function devServiceOption() {
+    message="Available Device Services to include in your compose file.\n
+            Press <SPACEBAR> to Select \n
+            Press <ENTER> to Skip \n
```

Comment:
This text is not being show for me.
![image](https://user-images.githubusercontent.com/44779287/164527886-6e4d0896-ccf0-4d65-ab3d-fe2c743e5222.png)

Reply:
It is for the App Services. I suspect it is a issue with screen space available in the small box. Can you make the box larger?